app/locale/en_US/template/email/contact_form_auto_reply.html

```diff
@@ -0,0 +1,11 @@
+<!--@subject Contact Form Confirmation @-->
+<!--@vars
+{"var data.name":"Sender Name",
+"var data.email":"Sender Email",
+"var data.telephone":"Sender Telephone",
+"var data.comment":"Comment"}
+@-->
+
+Thank you for contacting us, we will reply you shortly.
+<br>
+Have a nice day.
```

Comment:
I would evaluate if it is necessary to insert another paragraph for the situation when you receive the confirmation, without having used the contact form.

"In case you received this message without using the contact form on our website, please ignore it. Thank you for your understanding."

Reply:
@ADDISON74 I've also added this message to the template